docs/source/index.rst

```diff
@@ -31,7 +31,7 @@ From here you can set up your gateway with:
 
 Once the setup process has finished, you can then manage routes as follows:
 
-.. code-block:: console
+.. code-block:: bash
```

Comment:
Had a lexer warning for sphinx when this was set to console that was fixed by switching it to bash. Usually, I use console because it looks better with the GitHub markdown renderer but I guess on sphinx it may not be optimal